calico/common.py

```diff
@@ -271,6 +275,7 @@ def complete_logging(logfile=None,
                 handler.setLevel(file_level)
 
     # If we've been given a log file, log to file as well.
+
```

Comment:
There are several blank lines like this, seemingly added at random &ndash; presumably accidental?